docs/slides/population_structure/index.qmd

```diff
@@ -50,30 +49,19 @@ correlate with geographical distances
 
 :::: {.columns}
 ::: {.column width="49%"}
-
 ```{r}
 #| class-source: my_code_size_abitsmaller
 #| classes: my_code_size_abitsmaller
```

Comment:
Another thing: instead of using custom classes, you can change the font of the code block in the column div, or by using a fenced div around the code. So, either

```
::: {.column width="49%" .smaller}
```

or 

```
::: {.smaller}

```{r}
R code here
```

:::

I usually prefer the latter when it comes to columns, but either which is ok.